Classes/Backend/FileList.php

```diff
@@ -2,16 +2,12 @@
 namespace WebVision\WvFileDeleteReferences\Backend;
 
 /*
- * This file is part of the TYPO3 CMS project.
- *
- * It is free software; you can redistribute it and/or modify it under
- * the terms of the GNU General Public License, either version 2
- * of the License, or any later version.
+ * This file is part of the wv_file_delete_references Extension for TYPO3 CMS.
  *
  * For the full copyright and license information, please read the
  * LICENSE.txt file that was distributed with this source code.
```

Comment:
There is no `LICENCE.txt`. Either rename the `LICENSE` to `LICENSE.txt` or adjust he comment and remove `.txt`.